hadoop-client-modules/hadoop-client-runtime/pom.xml

```diff
@@ -159,6 +159,7 @@
                       <exclude>org.xerial.snappy:*</exclude>
                       <!-- leave out kotlin classes -->
                       <exclude>org.jetbrains.kotlin:*</exclude>
+                      <exclude>com.github.jnr:*</exclude>
```

Comment:
what breaks once this is excluded? the service stuff should be ok as it the irq stuff is only used server side AFAIK.